src/main/java/com/powsybl/network/conversion/server/NetworkConversionController.java

```diff
@@ -43,4 +43,21 @@
         NetworkInfos networkInfos = networkConversionService.importCase(caseUuid);
         return ResponseEntity.ok().body(networkInfos);
     }
+
+    @GetMapping(value = "/networks/{networkUuid}/{format}")
+    @ApiOperation(value = "Export a network from the network-store")
+    public ResponseEntity<ExportNetworkInfos> exportNetwork(@PathVariable("networkUuid") UUID networkUuid, @PathVariable("format") String format) {
```

Comment:
I think we should avoid sending the data (tens of megabytes) as base64 in a json object